scripts/pynag

```diff
@@ -690,6 +700,45 @@ def pretty_print(objects, attributes=None):
         pre = "-"*10
         post = "-"*(80-10-len(message))
         print pre + message + post
+
+def get_nagios_attributes(obj):
+    obj_attributes = {}
+    for attribute, value in obj.items():
+        # Don't return pynag internal attributes
+        # except the object type
+        if attribute == "id":
+            continue
+        if attribute == "meta":
+            obj_attributes["object_type"] = value['object_type']
+            continue
+        # or attributes with no value
+        if not value:
+            continue
+        obj_attributes[attribute] = value
+    return obj_attributes
+
+def print_kv(objects):
+    # If no column seperator was specified, use emptystring
+    if parser.values.seperator is None:
+        parser.values.seperator = ' '
+
+    for obj in objects:
+        obj_attributes = get_nagios_attributes(obj)
+        attribute_kv = ["{attribute}={value}".format(attribute=attribute, value=urllib.quote(value))
+                for attribute, value in obj_attributes.iteritems()]
+        print parser.values.seperator.join(attribute_kv)
+
+def print_json(objects):
+    # If no column seperator was specified, use emptystring
+    if parser.values.seperator is None:
+        parser.values.seperator = ' '
+
+    all_objects = []
+    for obj in objects:
+        obj_attributes = get_nagios_attributes(obj)
+        all_objects.append(obj_attributes)
+
+    print json.dumps(all_objects)
```

Comment:
You might consider adding intent=4 to the dumps call, to make it easier to read for humans.